config/routes.rb

```diff
@@ -3,4 +3,7 @@
     namespace :v1 do
     end
   end
+
+  get 'api/v1/organizations/public', to: 'organizations#show'
+
```

Comment:
You have to put this into the namespace, like this:
```suggestion
namespace :api, defaults: { format: 'json' } do
  namespace :v1 do
    get 'api/v1/organizations/public', to: 'organizations#show'
  end
end
```